x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/querydsl/query/SingleValueQueryTests.java

```diff
@@ -154,22 +152,24 @@ interface TestCase {
         void run(List<List<Object>> fieldValues, int count) throws IOException;
     }
 
-    private void runCase(List<List<Object>> fieldValues, int count, Integer docsStart, Integer docsStop) {
+    private void runCase(List<List<Object>> fieldValues, int count, Integer docsStart, Integer docsStop, boolean scanForMVs) {
```

Comment:
I probably should have put javadoc on this in the first pace, but I didn't. Now that you are adding the parameter could you add it? I think it'll help a lot.